sessionctx/variable/tidb_vars.go

```diff
@@ -832,5 +832,5 @@ type TopSQL struct {
 
 // TopSQLEnabled uses to check whether enabled the top SQL feature.
 func TopSQLEnabled() bool {
-	return TopSQLVariable.Enable.Load() && config.GetGlobalConfig().TopSQL.ReceiverAddress != ""
+	return TopSQLVariable.Enable.Load()
```

Comment:
If this variable is no longer controllable, how about moving it out to be a atomic.Bool instead?

Reply:
Not sure about what 'controllable' means.

For now, the report will modify it to indicate if someone subscribes to topsql data. Some components read it to determine whether to set tag and so on.

Reply:
By "controllable" I mean user can set it and *control* the behavior of the system. Variables are usually controllable things, while status like things are usually placed inside [`SHOW STATUS`](https://dev.mysql.com/doc/refman/8.0/en/show-status.html). I don't have strong opinion of where it can be placed or whether it should be ever known by the TiDB user. What I know is that variable may be not a good place now. 

cc @crazycs520

Reply:
We can move this variable into `topsql` pkg later.